packages/turbo-repository/js/README.md

```diff
@@ -0,0 +1,10 @@
+## Repositories
+
+Provides JS repository analysis functionality, scoped to the JS ecosystem.
```

Comment:
```suggestion
Provides repository analysis functionality, scoped to the web ecosystem.
```

Reply:
I don't think this is correct. `turbo` today works with the javascript ecosystem, whether or not that javascript is building something for the web. CLIs, backend services, etc. are all fair game. The scope is really anything that is driven via `package.json`.